components/multi-attribute-login/org.wso2.carbon.identity.multi.attribute.login.mgt/src/main/java/org/wso2/carbon/identity/multi/attribute/login/mgt/MultiAttributeLoginResolver.java

```diff
@@ -64,4 +64,16 @@ ResolvedUserResult resolveUser(String loginIdentifierValue, List<String> allowed
     AuthenticationResult authenticateWithIdentifier(String loginIdentifierValue, List<String> allowedAttributes,
                                                     Object credential, String tenantDomain);
 
+    /**
+     * Resolves possible users from given login identifier then returns resolved claim URI and user details if found a
+     * matching users.
+     *
+     * @param loginAttribute    User input when user try to login using multi attribute login.
+     * @param allowedAttributes List of claim URIs which are allowed by admin for multi attribute login.
+     * @param tenantDomain      User tenant domain.
+     * @return List<ResolvedUserResult> object with possible user details and resolved claim details.
+     */
+    List<ResolvedUserResult> resolvePossibleUsers(String loginAttribute, List<String> allowedAttributes,
```

Comment:
Don't we have to make this new method default, because others who have implemented this interface would get issues after merging this.